pkg/actions/anywhere/anywhere.go

```diff
@@ -0,0 +1,58 @@
+package anywhere
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/weaveworks/eksctl/pkg/version"
+)
+
+const (
+	// BinaryFileName defines the name of the anywhere binary.
+	BinaryFileName = "eksctl-anywhere"
+)
+
+// IsAnywhereCommand detects whether the user would like to execute anywhere based commands.
+func IsAnywhereCommand(args []string) (bool, error) {
+	if len(args) == 0 {
+		return false, nil
+	}
+
+	if args[0] == "anywhere" {
+		return true, nil
+	}
+
+	// if they have any args/flags before the anywhere command we should error
+	// e.g. eksctl --foo=bar anywhere should error
+	for _, arg := range args {
+		if arg == "anywhere" {
+			return false, fmt.Errorf("flags cannot be placed before the anywhere command")
+		}
+	}
+
+	return false, nil
+}
+
+// RunAnywhereCommand executes the anywhere binary.
+func RunAnywhereCommand(args []string) (int, error) {
+	if _, err := exec.LookPath(BinaryFileName); errors.Is(err, exec.ErrNotFound) {
+		return 1, fmt.Errorf(fmt.Sprintf("%q binary was not found on your path", BinaryFileName))
+	} else if err != nil {
+		return 1, fmt.Errorf("failed to lookup anywhere binary: %w", err)
```

Comment:
Do we want to refer to these as plugins rather than binaries? Not a big deal but just in terms of messaging to the customer as they probably only understand it as a plugin for eksctl right?
```suggestion
		return 1, fmt.Errorf(fmt.Sprintf("%q plugin was not found on your path", BinaryFileName))
	} else if err != nil {
		return 1, fmt.Errorf("failed to lookup anywhere plugin: %w", err)
```

Reply:
Sure!